spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -245,24 +245,6 @@
 				<scope>test</scope>
 			</dependency>
 
-			<dependency>
```

Comment:
I think that now it makes a lot more sense - just like the mock server removal, we remove client also. It has the same `<scope>test</scope>`